### PR TITLE
[WPML] Show venue detauls in List View

### PR DESCRIPTION
**Pull request by dgwatkins**
This issue was shared by Geoff in Slack.

https://lw.slack.com/archives/C01M100PPPU/p1668660044069739

Ticket: [ECP-1443](https://theeventscalendar.atlassian.net/browse/ECP-1443)

If I am not mistaken, we will need to check if custom tables are enabled for this to work.

[ECP-1443]: https://theeventscalendar.atlassian.net/browse/ECP-1443?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/Tribe/Integrations/WPML/Meta.php [hunk 1]
```diff
@@ -33,6 +34,8 @@ public function translate_post_id( $value, $object_id, $meta_key, $single ) {
 			return $value;
 		}
 
+		$object_id = Occurrence::normalize_id( $object_id );
```
Comment (lucatume) [inline src/Tribe/Integrations/WPML/Meta.php:37]: This class will be defined, but will work incorrectly when not in Custom Tables v1 context.
I would wrap this in a check for CT1 full activation:
```suggestion
         if( tribe()->getVar( 'ct1_fully_activated' ) ) {
		$object_id = Occurrence::normalize_id( $object_id );
	}
```

Comment (dgwatkins) [reply src/Tribe/Integrations/WPML/Meta.php]: Thanks, added.

**Comment by lucatume** [comment]
Changed the base to next release, updated the description to reference the Jira ticket.

**Comment by lucatume** [comment]
I'm closing this PR in favour of the up-to-date one: #4142

**Pull request closed without merging**